Resources/config/converters.xml

```diff
@@ -7,22 +7,22 @@
     <services>
         <service id="Sensio\Bundle\FrameworkExtraBundle\EventListener\ParamConverterListener" class="Sensio\Bundle\FrameworkExtraBundle\EventListener\ParamConverterListener" public="false">
             <tag name="kernel.event_subscriber" />
-            <argument type="service" id="Sensio\Bundle\FrameworkExtraBundle\Request\ParamConverter\ParamConverterManager" />
+            <argument type="service" id="sensio_framework_extra.converter.manager" />
             <argument>true</argument>
         </service>
 
-        <service id="Sensio\Bundle\FrameworkExtraBundle\Request\ParamConverter\ParamConverterManager" class="Sensio\Bundle\FrameworkExtraBundle\Request\ParamConverter\ParamConverterManager" />
+        <service id="sensio_framework_extra.converter.manager" class="Sensio\Bundle\FrameworkExtraBundle\Request\ParamConverter\ParamConverterManager" />
 
-        <service id="Sensio\Bundle\FrameworkExtraBundle\Request\ParamConverter\DoctrineParamConverter" class="Sensio\Bundle\FrameworkExtraBundle\Request\ParamConverter\DoctrineParamConverter" public="false">
+        <service id="framework_extra_bundle.doctrine_param_converter" class="Sensio\Bundle\FrameworkExtraBundle\Request\ParamConverter\DoctrineParamConverter" public="false">
             <tag name="request.param_converter" converter="doctrine.orm" />
             <argument type="service" id="doctrine" on-invalid="ignore" />
             <argument type="service" id="sensio_framework_extra.converter.doctrine.orm.expression_language" on-invalid="null" />
         </service>
 
-        <service id="Sensio\Bundle\FrameworkExtraBundle\Request\ParamConverter\DateTimeParamConverter" class="Sensio\Bundle\FrameworkExtraBundle\Request\ParamConverter\DateTimeParamConverter" public="false">
+        <service id="framework_extra_bundle.date_time_param_converter" class="Sensio\Bundle\FrameworkExtraBundle\Request\ParamConverter\DateTimeParamConverter" public="false">
             <tag name="request.param_converter" converter="datetime" />
         </service>
 
-        <service id="Symfony\Component\ExpressionLanguage\ExpressionLanguage" class="Symfony\Component\ExpressionLanguage\ExpressionLanguage" public="false" />
+        <service id="framework_extra_bundle.expression_language" class="Symfony\Component\ExpressionLanguage\ExpressionLanguage" public="false" />
```

Comment:
shouldn't it be `sensio_framework_extra.converter.doctrine.orm.expression_language` to match what is used in the `framework_extra_bundle.doctrine_param_converter` service ?